### PR TITLE
UNPLUGGED-828 | Restored initial screen error message

### PR DESCRIPTION
**Pull request by davidmolinacano**
Required by: https://github.com/doofinder/dfcommon/issues/828

After doing a huge refactor, this feature was removed accidentally (https://github.com/doofinder/doofinder-prestashop/commit/31d8a242e8210a6de647b066c00eea0cc221667a?diff=split), so it has been restored again.

image

### CODE REVIEW

File: doofinder.php [hunk 1]
```diff
@@ -31,9 +31,11 @@ class Doofinder extends Module
     public $ps_layered_full_tree = true;
     public $searchBanner = false;
 
+    // Feel free to change this value to your own local env or ngrok
+    const DOOMANAGER_URL = 'https://admin.doofinder.com';
```
Comment (albertodoofinder) [inline doofinder.php:35]: eu1 and us1 regions are compatible with this url?

Comment (davidmolinacano) [reply doofinder.php:35]: In the commit where this part was deleted it was using this one, but I'll try to research about it:
image

Comment (davidmolinacano) [reply doofinder.php:35]: Thank you for the feedback :smiley:

Comment (davidmolinacano) [reply doofinder.php:35]: I was able to open both of them without any issue

Comment (davidmolinacano) [reply doofinder.php:35]: Explanation of why this works:

When a user accesses, he is redirected to a region depending on the latency he has, that is, when he is accessing from Spain, he is redirected to the EU region, which is the one with the lowest response time.

The URL that you have configured is simply to check if the client's Prestashop can connect with Doofinder regardless of the region. In the end, what it does is simply a ping to the Doofinder URL, so we don't care if it goes to one region or another.

**Review by albertodoofinder — APPROVED** [review]

File: doofinder.php (hunk 1 above)
Comment (albertodoofinder) [reply doofinder.php:35]: Thanks @davidmolinacano for the explanation

Comment (davidmolinacano) [reply doofinder.php:35]: @ivan-doofinder has explained it to me with this level of detail, so he deserves the credit :clap:

**Review by Aitor-Corrales — APPROVED** [review]

**Review by ivan-doofinder — APPROVED** [review]

**Pull request merged**